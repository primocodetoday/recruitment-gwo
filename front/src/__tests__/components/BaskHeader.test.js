import React from 'react';
import { render } from '@testing-library/react';
import { BasketHeader } from 'components';

it('<BasketHeader/> render properly', () => {
  const { getByText, container } = render(<BasketHeader />);
  expect(getByText('tytuł')).toBeTruthy();
  expect(getByText('szt.')).toBeTruthy();
  expect(getByText('cena')).toBeTruthy();
  expect(container.firstChild).toMatchSnapshot();
});
