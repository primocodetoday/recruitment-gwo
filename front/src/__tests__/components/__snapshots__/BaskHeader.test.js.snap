// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BasketHeader/> render properly 1`] = `
<div
  class="mb-2 px-3 text-muted text-uppercase text-center align-items-center row"
>
  <div
    class="border-right text-left pl-2 col-md-8 col-sm-6 col-5"
  >
    <p
      class="h6"
    >
      tytu≈Ç
    </p>
  </div>
  <div
    class="border-right px-0 col-md-1 col-sm-2 col-2"
  >
    <p
      class="h6"
    >
      szt.
    </p>
  </div>
  <div
    class="border-right px-0 col-md-2 col-sm-2 col-3"
  >
    <p
      class="h6"
    >
      cena
    </p>
  </div>
  <div
    class="text-center col-md-1 col-sm-2 col-2"
  >
    <i
      class="fas fa-trash"
    />
  </div>
</div>
`;
